airflow/www/views.py

```diff
@@ -337,6 +337,9 @@ def get_int_arg(value, default=0):
 
         num_of_pages = int(math.ceil(num_of_all_dags / float(dags_per_page)))
 
+        state_color_mapping = State.state_color.copy()
+        state_color_mapping["null"] = state_color_mapping.pop(None)
```

Comment:
`None` is one of the keys in `State.state_color`, and it cannot be handled by Flask's `tojson` directly. So a special treatment is needed here.

A related issue I raised in Flask: https://github.com/pallets/flask/issues/3599

Reply:
Keys in JSON must be strings, so `{ null: "x"}` isn't valid JSON. So it _sort_ of makes sense that it doesn't get added. Sort of.

Reply:
Python already handles it for us:
```python
import json
print(json.dumps({None: 123}))
```
gives us `{"null": 123}`.

The issue is the `sort_keys` argument when we run `json.dumps()`.